### DISCUSSION TITLE
Osu lazer Android cant input any Characters.

### DISCUSSION THREAD

**Discussion by IntenseScoutHD** [ppy/osu#15926] [Q&A] [answered]
I have just downloade the apk and started the game. It instantly wanted me to sign in or register(normal), so i wanted to create a account but it didnt work. I wrote but it didnt showed the letters on screen. Help is highly needed pls.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/search?q=android+keyboard&type=discussions
